greengrocer_class.rb

```diff
@@ -9,23 +9,22 @@ def initialize(name:, price:)
 end
 
 class Greengrocer
-  attr_reader :product_params
-  def initialize(product_params)
-    @product_params = []
-    register_product(product_params)
+  attr_reader :products
+  def initialize(products)
```

Comment:
インスタンス変数 `@products` はProductのインスタンスの配列ですので `@product_params` は不自然ですが，ここの引数はパラメータの配列ですので，`product_params`のままにすべきです。

```suggestion
  def initialize(product_params)
```

Reply:
ご確認ありがとうございます。修正いたします。